### PR TITLE
[4.0] remove file check

### PR DESCRIPTION
**Pull request by brianteeman**
Pull Request for Issue #35036 

### Summary of Changes
Remove the check for if the file exists before inserting an image

### Testing Instructions
Create  an image with a space in the name
Create a custom field for an article of type media
Create an article and use the image in the custom field and the full article image

### Actual result BEFORE applying this Pull Request
Only the full article image is displayed

### Expected result AFTER applying this Pull Request
Both the full article image and the media field image are displayed

### Note
The reason that a file with a space in the name works everywhere else except for the media field is that the media field is the only place that has the file exists check

### CODE REVIEW

File: plugins/fields/media/tmpl/media.php [hunk 1]
```diff
@@ -26,31 +26,29 @@
 
 if ($value)
 {
+
```
Comment (richard67) [inline plugins/fields/media/tmpl/media.php]:
```suggestion
```
I think this empty line is not required by our code style and even wrong, even if it doesn't make PHPCS fail in drone.

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on 4d4363b2d94e5be66112541a0ea8d1a946d989ad<br /><br />Thanks!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35038">issues.joomla.org/tracker/joomla-cms/35038</a>.</sub>

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 4d4363b2d94e5be66112541a0ea8d1a946d989ad<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35038">issues.joomla.org/tracker/joomla-cms/35038</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35038">issues.joomla.org/tracker/joomla-cms/35038</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks George

**Pull request merged**